worlds/hk/__init__.py

```diff
@@ -659,6 +659,8 @@ class HKItem(Item):
     def __init__(self, name, advancement, code, type: str, player: int = None):
         if name == "Mimic_Grub":
             classification = ItemClassification.trap
+        elif name == "Godtuner":
```

Comment:
I believe this should not be needed because godtuner is in the keys pool which would make it be progression inherently.

Reply:
keys pool isn't called out in that init, and if they are not called out they fall back into presence in ExtractedData.py logic_items
reason it happens for world sense is the Dreamer group is called out to set to prog skip balancing

Reply:
confirmed with a python shell
```
import Generate
multi = Generate.main()
x = [item for item in multi.itempool if item.name =="Godtuner"][0]
x.classification
```
was filler on main, prog on this branch

Reply:
Do simple keys also not count as prog then? Because that would be a problem also

Reply:
ah, 'advancement' is also set if the item is in item_effects which simple_key is, didn't see that at first but still no mention of godtuner in there either

Reply:
Interesting, guess this is fine